art/defences/preprocessor/preprocessor.py

```diff
@@ -162,6 +164,55 @@ def estimate_forward(self, x: "torch.Tensor", y: Optional["torch.Tensor"] = None
         """
         raise NotImplementedError
 
+    def __call__(self, x: np.ndarray, y: Optional[np.ndarray] = None) -> Tuple[np.ndarray, Optional[np.ndarray]]:
+        """
+        Apply local spatial smoothing to sample `x`.
```

Comment:
Line 169-173: I think the documents are not updated. It is for label smoothing.

Reply:
I agree, fixed with 1d7b82217804353137a7e9c916459f8641c80632